ghcide/src/Development/IDE/Core/Compile.hs

```diff
@@ -902,8 +902,8 @@ indexHieFile se mod_summary srcPath !hash hf = do
             Just pendingHash -> pendingHash /= hash
         unless newerScheduled $ do
           pre optProgressStyle
-          withHieDb (\db -> HieDb.addRefsFromLoaded db targetPath (HieDb.RealFile $ fromNormalizedFilePath srcPath) hash hf')
-          post
+          bracket_ (pre optProgressStyle) post $
+            withHieDb (\db -> HieDb.addRefsFromLoaded db targetPath (HieDb.RealFile $ fromNormalizedFilePath srcPath) hash hf')
```

Comment:
Is the first line still necessary?

Reply:
OMG, that's a mistake. Sorry. Yes. I'll remove it.